platform/src/main/java/org/stellar/anchor/platform/service/Sep6MoreInfoUrlConstructor.java

```diff
@@ -0,0 +1,68 @@
+package org.stellar.anchor.platform.service;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.SneakyThrows;
+import org.apache.http.client.utils.URIBuilder;
+import org.stellar.anchor.SepTransaction;
+import org.stellar.anchor.auth.JwtService;
+import org.stellar.anchor.auth.Sep6MoreInfoUrlJwt;
+import org.stellar.anchor.platform.config.MoreInfoUrlConfig;
+import org.stellar.anchor.platform.config.PropertyClientsConfig;
+import org.stellar.anchor.sep24.MoreInfoUrlConstructor;
+import org.stellar.anchor.sep6.Sep6Transaction;
+import org.stellar.anchor.util.ConfigHelper;
+
+public class Sep6MoreInfoUrlConstructor extends MoreInfoUrlConstructor {
+  PropertyClientsConfig clientsConfig;
+  private final MoreInfoUrlConfig config;
+  private final JwtService jwtService;
+
+  public Sep6MoreInfoUrlConstructor(
+      PropertyClientsConfig clientsConfig, MoreInfoUrlConfig config, JwtService jwtService) {
+    this.clientsConfig = clientsConfig;
+    this.config = config;
+    this.jwtService = jwtService;
+  }
+
+  @SneakyThrows
```

Comment:
Do we need `@SneakyThrows` here if the called `construct` doesn't throw checked exceptions?